aptos-move/aptos-aggregator/src/resolver.rs

```diff
@@ -203,7 +203,7 @@ pub trait TDelayedFieldView {
         &self,
         delayed_write_set_ids: &HashSet<Self::Identifier>,
         skip: &HashSet<Self::ResourceKey>,
-    ) -> Result<BTreeMap<Self::ResourceKey, (StateValueMetadata, u64)>, PanicError>;
+    ) -> PartialVMResult<BTreeMap<Self::ResourceKey, (StateValueMetadata, u64)>>;
```

Comment:
let's comment / document why the return type is different from get_reads_needing_exchange?